### DISCUSSION TITLE
求助：主笔记按钮突然不见了

### DISCUSSION THREAD

**Discussion by shuangwang-zoe** [windingwind/zotero-better-notes#491] [General]
今天突然发现主笔记按钮不见了（如下图），此外之前设置好的从注释自动添加到笔记功能也突然无法使用，并且edit里面也没有这个选项了。重装了最新版本的better notes也还是无法解决。请问一下大家有出现类似的情况吗？谢谢！
![image](https://github.com/windingwind/zotero-better-notes/assets/69445575/2180d7a5-df79-43ca-93f0-a1a0bdb79b62)

**Comment by windingwind** [comment]
所有功能移到右侧齿轮
